File: selection sort..c
#include<stdio.h>
main()
{
int n;
printf("please enter the value of n");
scanf("%d",&n);
int i,j,k,a[n];
printf("please enter the numbers");
for(i=0;i<n;i++)
{
	scanf("%d",&a[i]);
}
for(i=0;i<n;i++)
{
	for(j=i+1;j<n;j++)
	{
		if(a[i]>a[j])
		{
			k=a[i];
			a[i]=a[j];
			a[j]=k;
		}
	}
}
printf("elements in sorted order are");
for(i=0;i<n;i++)
{
	printf("%d",a[i]);
	printf("\n");
}
}

File: selection sort.py
n=int(input("enter the value of n"))
a=[]
print("enter the list elements")
for i in range(0,n):
    k=int(input())
    a.append(k)
for i in range(0,n):
    for j in range(i+1,n):
        if(a[i]>a[j]):
            c=a[i]
            a[i]=a[j]
            a[j]=c
print("elements in sorted order are")
print(a)
    
            
    
    
    
